### PR TITLE
Fix Foxnews and Foxbusiness video playback

### PR DESCRIPTION
**Pull request by ryanbr**
Video playback was broken, this will fix it. Existing in Easyprivacy, but Disconnect was overruling it.

Included a Tracking /ads.js whitelist to track users with.

### CODE REVIEW

**Comment by ryanbr** [comment]
Correction, block not needed.

File: brave-unbreak.txt [hunk 1]
```diff
@@ -189,6 +189,10 @@
 @@||googletagservices.com/tag/js/gpt.js$script,domain=spiegel.de
 @@||mxcdn.net/bb-mx/$script,domain=spiegel.de
 @@||imagesrv.adition.com/banners/$image,domain=spiegel.de
+! Fix foxnews video playback
+@@||fncstatic.com/static/isa/app/lib/VisitorAPI.js$script,domain=foxbusiness.com|foxnews.com
+! Adblock-Tracking: foxnews.com / foxbusiness.com
+@@||fncstatic.com/static/v/all/js/ads.js$script,domain=foxbusiness.com|foxnews.com
```
Comment (bbondy) [inline brave-unbreak.txt]: If disconnect is breaking this ONLY, then you need to make the adblock lib search for an exception. To do that.
So I think you need to duplicate these rules with both block and exceptions.

! Fix foxnews video playback
||fncstatic.com/static/isa/app/lib/VisitorAPI.js$script,domain=foxbusiness.com|foxnews.com
@@||fncstatic.com/static/isa/app/lib/VisitorAPI.js$script,domain=foxbusiness.com|foxnews.com
! Adblock-Tracking: foxnews.com / foxbusiness.com
||fncstatic.com/static/v/all/js/ads.js$script,domain=foxbusiness.com|foxnews.com
@@||fncstatic.com/static/v/all/js/ads.js$script,domain=foxbusiness.com|foxnews.com

Comment (snyderp) [reply brave-unbreak.txt]: @bbondy I don't see fncstatic.com in the tracking protection data

https://github.com/brave/tracking-protection/blob/master/data/disconnect.json

My understanding was that if the domains not in there, the tracking protection is def not involved.  Is that incorrect?

Comment (ryanbr) [reply brave-unbreak.txt]: Caused by;
```
easyprivacy_general.txt:/VisitorAPI.js
```
And whiteslist in EP:
```
easyprivacy_whitelist.txt:@@||fncstatic.com^*/visitorapi.js$script,domain=foxbusiness.com|foxnews.com
```

Comment (snyderp) [reply brave-unbreak.txt]: Yes, only referring to the block and then allow pattern.  That pattern is specifically only needed when working around tracking protection.

Comment (ryanbr) [reply brave-unbreak.txt]: So is it safe to merge? no blocking filter needed because its specific to EP only and not the disconnect list

**Pull request merged**